pkg/operators/ebpf/ebpf.go

```diff
@@ -676,6 +686,34 @@ func (i *ebpfInstance) Start(gadgetCtx operators.GadgetContext) error {
 	}
 	i.collection = collection
 
+	if i.bpfOperator.gadgetObjs != nil {
+		// collect program IDs and map IDs for this gadget
+		i.bpfOperator.mu.Lock()
+		gadgetObjs := gadgetObjs{}
+
+		for _, p := range i.collection.Programs {
+			info, err := p.Info()
+			if err != nil {
+				continue
```

Comment:
Log a warning this failed? I think it's an important log because (1) user enabled stats on daemon and (2) user requested to collect stats for this gadget (see previous comment), and we will not provide info about that specific program. I think at least a warn log is worth it.